src/views/HideoutTable.vue

```diff
@@ -80,21 +82,33 @@
         this.currentModules = []
         this.lockedModules = []
         var tempHideout = this.hideoutDataDefault.modules
+
+        if (this.$store.get('progress/dataVersion') < 2) {
```

Comment:
👍 Will just leave this in for the next couple months or until next wipe (will try to force a data wipe then, for this and a couple other schema changes I expect to make)